app/controllers/laa_references_controller.rb

```diff
@@ -80,13 +80,41 @@ def link_attempt_params
   end
 
   def resource
-    CourtDataAdaptor::Resource::LaaReference
+    if Feature.enabled?(:laa_references)
+      logger.info 'USING_V2_ENDPOINT'
+      LaaReferences
+    else
+      logger.info 'USING_V1_ENDPOINT'
+      CourtDataAdaptor::Resource::LaaReference
+    end
   end
 
   def resource_params
     @resource_params ||= @link_attempt.to_link_attributes
   end
 
+  def resource_save
+    if Feature.enabled?(:laa_references)
+      begin
+        logger.info 'CALLING_V2_MAAT_LINK'
+        @laa_reference.save!
```

Comment:
a shebang will mean activeresource validations always run before the call is sent out. Although, we don't have validations on laa_reference...should we? 

I don't think using save! is a problem because at least it says we always run validations before sending to the api in our app... it's easy to miss the difference between save and save! so using save! everywhere protects us from missing validations...
i don't mind if it's save! everywhere instead of patches of save and save! throughout our app
 idk, this is a stream of thought i'm having now sorry